playground.go

```diff
@@ -0,0 +1,384 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/appwrite/sdk-for-go/account"
+	"github.com/appwrite/sdk-for-go/appwrite"
+	"github.com/appwrite/sdk-for-go/client"
+	"github.com/appwrite/sdk-for-go/databases"
+	"github.com/appwrite/sdk-for-go/functions"
+	"github.com/appwrite/sdk-for-go/id"
+	"github.com/appwrite/sdk-for-go/permission"
+	"github.com/appwrite/sdk-for-go/storage"
+	"github.com/appwrite/sdk-for-go/users"
+)
+
+var appwriteProject = "<PROJECT_ID>"
+var appwriteApiKey = "<API_KEY>"
+
+var (
+	appwriteClient   client.Client
+	appwriteDatabase *databases.Databases
+	appwriteFunction *functions.Functions
+	appwriteStorage  *storage.Storage
+	appwriteAccount  *account.Account
+	appwriteUsers    *users.Users
+	databaseId       string
+	collectionId     string
+	documentId       string
+	userId           string
+	bucketId         string
+	fileId           string
+	functionId       string
+)
+
+func main() {
+	appwriteClient = appwrite.NewClient(
+		appwrite.WithProject(appwriteProject),
+		appwrite.WithKey(appwriteApiKey),
+		// appwrite.WithJWT("JWT") // Use this to authenticate with JWT instead of API_KEY
+	)
+
+	appwriteDatabase = appwrite.NewDatabases(appwriteClient)
+	appwriteFunction = appwrite.NewFunctions(appwriteClient)
+	appwriteUsers = appwrite.NewUsers(appwriteClient)
+	appwriteStorage = appwrite.NewStorage(appwriteClient)
+	appwriteAccount = appwrite.NewAccount(appwriteClient)
+
+	// GetAccount() // Use this only with JWT
+	CreateUser()
+	ListUsers()
+	DeleteUser()
+
+	CreateDatabase()
+
+	CreateCollection()
+	ListCollection()
+
+	CreateDocument()
+	ListDocuments()
+
+	DeleteDocument()
+	DeleteCollection()
+	DeleteDatabase()
+	CreateBucket()
+	ListBuckets()
+
+	// UploadFile() TODO: Fix how we send content range etc in SDK Go
+	ListFiles()
+	DeleteFile()
+
+	DeleteBucket()
```

Comment:
What's the thought process behind the line breaks?

Reply:
The line breaks were to match the consistency across the other existing playgrounds, as well as separate the different products and services from each other.
- [playground-python](https://github.com/appwrite/playground-for-python/blob/master/playground.py)
- [playground-ruby](https://github.com/appwrite/playground-for-ruby/blob/main/lib/playground.rb)
- [playround-node](https://github.com/appwrite/playground-for-node/blob/master/src/app.js)

Reply:
Grouping by products would be good.